src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -1091,6 +1091,37 @@ public void GetAsync_ServerNeedsAuthAndNoCredential_StatusCodeUnauthorized()
         }
 
         [OuterLoop] // TODO: Issue #11345
+        [Theory]
+        [InlineData("")] // missing size
+        [InlineData("10000000000000000")] // overflowing size
+        public async Task GetAsync_InvalidChunkSize_ThrowsHttpRequestException(string chunkSize)
+        {
+            await LoopbackServer.CreateServerAsync(async (server, url) =>
+            {
+                using (var client = new HttpClient())
+                {
+                    string partialResponse = "HTTP/1.1 200 OK\r\n" +
+                        "Transfer-Encoding: chunked\r\n" +
```

Comment:
Best practice is to always have a Date: response header in here.  I believe this is a MUST per RFC.  The other HTTP tests that use loopback server do that.